cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/workflow/DefaultWorkflowConditionConfigurer.java

```diff
@@ -42,8 +45,10 @@
   private List<WorkflowNode> currentBranch;
   private boolean addingToIfBranch = true;
   private final String predicateClassName;
+  private final String uniqueName;
```

Comment:
nit: because there are method parameters in this class called ``uniqueName``, is it better to qualify this as ``conditionUniqueName``?